frontend/src/static/css/includes/form_controls/_mixins_form_controls.scss

```
@@ -1,15 +1,17 @@
 // @param {String} $string - Initial string
 // @param {String} $search - Substring to replace
 // @param {String} $replace ('') - New value
 // @return {String} - Updated string
 @function str-replace($string, $search, $replace: '') {
-	$index: str-index($string, $search);
+	$index: string.index($string, $search);
 
 	@if $index {
-		@return str-slice($string, 1, $index - 1) + $replace +
-			str-replace(str-slice($string, $index + str-length($search)), $search, $replace);
+		@return string.slice($string, 1, $index - 1) + $replace +
```

Stylelint (17.11.1)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)